tests/pkg_libcose/Makefile

```
@@ -0,0 +1,20 @@
+include ../Makefile.tests_common
+
+# HACL* only compiles on 32bit platforms
+BOARD_BLACKLIST := arduino-duemilanove arduino-mega2560 arduino-uno \
+                   jiminy-mega256rfr2 mega-xplained waspmote-pro \
+                   chronos msb-430 msb-430h telosb wsn430-v1_3b wsn430-v1_4 z1
+
+BOARD_INSUFFICIENT_MEMORY := nucleo-f030r8 nucleo-f031k6 nucleo-f042k6 \
+                             nucleo-l031k6 nucleo-l053r8 stm32f0discovery
+
+TEST_ON_CI_WHITELIST += all
```

> > But the question was if we want to introduce a **2min** wait-time to the CI tests, because this test requires two minutes to finish on samr21-xpro.
> 
> ah that question: well, I would say no don't run the tests on hardware if it takes that long

@bergzand then please change this

```suggestion
TEST_ON_CI_WHITELIST += native
```

---

Good idea! Applied